.env

```diff
@@ -0,0 +1,26 @@
+# This file defines all environment variables that the application needs.
```

Comment:
as uz ..env.dist failo pushinima i repo
.env geriau pasidaryti is disto paciam ir ten susimesti savo credentialus, problema bus jai susidesiu savo ir pridesiu dar koki vienapapildoma secreta , noredamas supushinti man reiks nepamirsti nusiresetinti savus i defaultinius

Reply:
bet tai nereikia nusiresetint .env prieš pushą. Tu turėtum pasikurt `.env.local` ir ten laikyt savo credentials (šitas failas nepushinamas, bet užloadinamas)